jib-core/src/main/java/com/google/cloud/tools/jib/frontend/JavaDockerContextGenerator.java

```diff
@@ -131,6 +131,11 @@ private static String mapToDockerfileString(Map<String, String> map, String comm
     return joiner.toString();
   }
 
+  @VisibleForTesting
+  static String getUnixPath(Path path) {
```

Comment:
I think another PR has a similar utility, might be worth using the same for both?

Reply:
Certainly. Since that utility is not yet available, I'll add a TODO here to use the utility.